config/arpwatch.xml

```diff
@@ -81,9 +81,12 @@
 	</additional_files_needed>
 	<fields>
 		<field>
-			<fielddescr>Listening Interface</fielddescr>
-			<fieldname>interface</fieldname>
-			<description>Choose the desired listening interface here.</description>
+			<fielddescr>Interfaces</fielddescr>
+			<fieldname>interface_array</fieldname>
```

Comment:
It would be nice if the interface that was already set by user before upgrade stay set after